_locales/en/messages.json

```diff
@@ -344,16 +344,16 @@
     "message": "No results found in \"$1\". Perhaps try different keywords?"
   },
   "captureCommentError": {
-    "message": "Message added by Scratch Addons: it appears that your comment doesn't follow $1, which asks Scratchers not to advertise or name browser extensions. This comment wasn't posted because it will probably result in your account getting muted from commenting for 5 minutes."
+    "message": "Message added by Scratch Addons: it appears that your comment doesn't follow $1, which asks Scratchers not to advertise or name browser extensions for safety reasons. This comment wasn't posted because mentioning the name of browser extensions violates the aforementioned policy."
```

Comment:
So, then how about...?
```suggestion
    "message": "Message added by Scratch Addons: it appears that your comment doesn't follow $1, which asks Scratchers not to advertise or name browser extensions for safety reasons. This comment wasn't posted because it would likely result in your account being temporarily muted."
```

Reply:
Could this be interpreted as our extension being unsafe?

Reply:
@Samq64 Possibly it could.

Reply:
The other parts of that message might be interpreted like that too.